libobs/obs-windows.c

```diff
@@ -64,13 +64,42 @@ char *find_libobs_data_file(const char *file)
 	struct dstr path;
 	dstr_init(&path);
 
+	WCHAR *env_path = 0;
+	DWORD env_path_sz = 
+		GetEnvironmentVariableW(L"OBS_DATA_PATH", NULL, 0);
```

Comment:
These two lines here would fit under the 80 column guideline if combined.  They're unnecessarily split.